android/src/test/java/com/marigold/rnsdk/RNMessageStreamModuleTest.kt

```diff
@@ -63,6 +63,20 @@ class RNMessageStreamModuleTest {
         messageStream = staticMessageStream.constructed()[0]
     }
 
```

Comment:
Can we also add a test for the `acknowledgeEvent` flow to ensure that the value provided is returned?